indico/modules/events/abstracts/controllers/management.py

```diff
@@ -279,3 +282,46 @@ def _process(self):
         sorted_abstracts = sorted(self.abstracts, key=attrgetter('friendly_id'))
         pdf = ConfManagerAbstractsToPDF(self.event_new, sorted_abstracts)
         return send_file('abstracts.pdf', pdf.generate(), 'application/pdf')
+
+
+class RHManageReviewingRoles(RHManageAbstractsBase):
+    """Configure track roles (reviewers/conveners)."""
+
+    def _process(self):
+        roles = get_roles_for_event(self.event_new)
+        form = AbstractReviewingRolesForm(event=self.event_new, obj=FormDefaults(roles=roles))
+        conveners = set()
+        reviewers = set()
+
+        if form.validate_on_submit():
+            # Handle global reviewers/conveners
+            if '' in form.roles.data:
+                role_data = form.roles.data.pop('')
+                self.event_new.global_conveners = set(User.find(User.id.in_(role_data['convener'])))
+                self.event_new.abstract_global_reviewers = set(User.find(User.id.in_(role_data['reviewer'])))
+                conveners |= self.event_new.global_conveners
+                reviewers |= self.event_new.abstract_global_reviewers
+
+            track_dict = {track.id: track for track in Track.find(Track.id.in_(form.roles.data))}
```

Comment:
Why not query only tracks for the event? Then you don't need the check whether it's from the same event below...